lib/model/appConfig/app_config_model.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:mobx/mobx.dart';
+
+part 'app_config_model.g.dart';
+
+class AppConfigModel = _AppConfigModel with _$AppConfigModel;
+
+abstract class _AppConfigModel with Store {
+
+  @observable
+  bool displayBalance = true;
```

Comment:
to be protective we can set the value to false by default and then it will be set from storage in ctrl